dist/templates/ovn-setup.yaml.j2

```diff
@@ -132,6 +142,7 @@ subjects:
   name: ovn
   namespace: ovn-kubernetes
 
+{% if after_initial_setup != "true" -%}
```

Comment:
I am not clear on the use of `after_initial_setup` and when it can be set to true ? so the idea is avoid recreating the namespaces ?

Reply:
See https://github.com/ovn-org/ovn-kubernetes/pull/3001#discussion_r899244802